lib/package-generator-view.coffee

```diff
@@ -89,6 +97,9 @@ class PackageGeneratorView extends View
 
     @runCommand(atom.packages.getApmPath(), args, callback)
 
+  isInPackageMode: ->
```

Comment:
Yeah that's what it was [here](https://github.com/atom/package-generator/pull/36/files#diff-5efa39df2af950a8f497782bc40d7625L37) but since we're doing the [same check](https://github.com/atom/package-generator/pull/36/files#diff-5efa39df2af950a8f497782bc40d7625R84) in two places now, a method for the check seems better than a constant or string duplication to me.